.gitignore

```diff
@@ -16,8 +16,8 @@ bin/
 
 # Binaries are copied (as needed) to the same location as their respective Dockerfile in
 # order to simplify docker build commands. Ignore these files if proper clean up fails.
-cmd/ocagent/ocagent_linux
 cmd/occollector/occollector_linux
+cmd/otelsvc/otelsvc_linux
```

Comment:
Not for this PR but just crossed my mind that it is an opportunity to perhaps not use the $(GOOS) as suffix. Something like: by default we build otelsvc and separate by folder, eg.: ./bin/linux/otelsvc - a minor thing but to decide before first release.

Reply:
I had the same thought. It looks odd to have _linux in the executable name.